packages/react-core/src/components/DataList/DataListContent.tsx

```diff
@@ -14,7 +14,7 @@ export interface DataListContentProps extends React.HTMLProps<HTMLElement> {
   /** Flag to show if the expanded content of the DataList item is visible */
   isHidden?: boolean;
   /** Flag to remove padding from the expandable content */
-  noPadding?: boolean;
+  hasNoPadding?: boolean;
```

Comment:
Can we change this to `hasPadding` and default it to `true`? Sounds a little better and codemods fix it all the same.

Reply:
@tlabaj Thoughts?

I don't mind either way.

Reply:
Keeping as `hasNoPadding` per standup discussion for now